site/www/vendors/vanta/viz.min.js

```diff
@@ -1 +1 @@
Caster.setFromCamera(new THREE.Vector2(this.mouse.x,this.mouse.y),this.camera),this.animationLoop())}genPoint(t,i,e){const o=new THREE.SphereGeometry(.2,12,12),s=new THREE.MeshLambertMaterial({color:this.options.color,transparent:!0,opacity:.4}),n=new THREE.Mesh(o,s);return n.position.set(t,i,e),n.r=25e-5*(4*Math.random()-2),n}init(){const t=new THREE.Group;t.position.set(0,0,0);let{points:i,spacing:e}=this.options;const o=i*i*2;this.linePositions=new Float32Array(o*o*3),this.lineColors=new Float32Array(o*o*3);const s=new THREE.BufferGeometry;s.setAttribute("position",new THREE.BufferAttribute(this.linePositions,3)),s.setAttribute("color",new THREE.BufferAttribute(this.lineColors,3)),s.computeBoundingSphere(),s.setDrawRange(0,0);const n=new THREE.LineBasicMaterial({vertexColors:THREE.VertexColors});n.linewidth=.25,this.linesMesh=new THREE.LineSegments(s,n),this.linesMesh.renderOrder=2,t.add(this.linesMesh);for(let o=0;o<=i;o++)for(let s=0;s<=i;s++){const n=ri(-3,3),r=(o-i/2)*e+ri(-5,5);let h=(s-i/2)*e+ri(-5,5);o%2&&(h+=.5*e);const l=this.genPoint(r,n-ri(5,15),h),a=this.genPoint(r+ri(-5,5),n+ri(5,15),h+ri(-5,5));t.add(l,a),this.points.push(l,a)}this.renderer.setSize(this.el.offsetWidth,this.el.offsetHeight),this.renderer.setPixelRatio(window.devicePixelRatio),this.camera=new THREE.PerspectiveCamera(25,this.el.offsetWidth/this.el.offsetHeight,.01,1e4),this.camera.position.set(50,100,150),this.camera.lookAt(0,0,0),this.scene=new THREE.Scene,this.scene.add(this.camera),this.scene.add(new THREE.AmbientLight(16777215,.75)),this.scene.add(t)}onUpdate(){let t,i,e,o,s,n,r=0,h=0,l=0,a=0;this.updatedCountXX+=1;for(let u=0;u<this.points.length;u++){o=this.points[u],this.rayCaster&&(this.mouse.updated?(i=.25*(12-this.rayCaster.ray.distanceToPoint(o.position)))>1?(a=1,o.material.color=this.highlightColor):(a=0,o.material.color=this.options.color):o.material.color!==this.options.color&&(o.material.color=this.options.color)),0!==o.r&&(n=Math.atan2(o.position.z,o.position.x)+o.r,t=Math.sqrt(o.position.z**2+o.position.x**2),o.position.x=t*Math.cos(n),o.position.z=t*Math.sin(n));for(let i=u;i<this.points.length;i++)if(s=this.points[i],!((t=Math.sqrt((o.position.x-s.position.x)**2+(o.position.y-s.position.y)**2+(o.position.z-s.position.z)**2))>=this.options.maxDistance)){if(a)e=this.highlightColor;else{let i=1-t/this.options.maxDistance;i<0?i=0:i>1&&(i=1),e=this.options.backgroundColor.clone().lerp(this.options.color,i)}this.linePositions[r++]=o.position.x,this.linePositions[r++]=o.position.y,this.linePositions[r++]=o.position.z,this.linePositions[r++]=s.position.x,this.linePositions[r++]=s.position.y,this.linePositions[r++]=s.position.z,this.lineColors[h++]=e.r,this.lineColors[h++]=e.g,this.lineColors[h++]=e.b,this.lineColors[h++]=e.r,this.lineColors[h++]=e.g,this.lineColors[h++]=e.b,l++}}this.linesMesh.geometry.setDrawRange(0,2*l),this.linesMesh.geometry.attributes.position.needsUpdate=!0,this.linesMesh.geometry.attributes.color.needsUpdate=!0}}export default Viz;
```

Comment:
this is just the .4 opacity change; minification should be moved to build step, in the future.